__tests__/app.test.js

```diff
@@ -29,4 +29,24 @@ describe("GET: /api/topics", () => {
             expect(response.status).toBe(404)
         })
     })
-});
\ No newline at end of file
+});
+
+describe("GET: /api/articles/:article_id", () => {
+    test("200: Responds with an article object with the correct keys", () => {
+        return request(app)
+        .get("/api/articles/1")
+        .expect(200)
+        .then((response) => {
+            expect(response.body.article)
+            .toHaveProperty('author', 'title', 'article_id', 'body', 'topic', 'created_at', 'votes', 'article_img_url')
+        })
+    })
+    test('GET:404 sends an appropriate and error message when given a valid but non-existent id', () => {
+        return request(app)
+          .get('/api/articles/99')
+          .expect(404)
+          .then((response) => {
+            expect(response.body.msg).toBe('Not found');
+        })
+    })
```

Comment:
You're missing a test for an invalid input such as if I went to /api/articles/banana what would happen? What error code and message could I expect?